lib/need_for_type/display_window.rb

```diff
@@ -36,44 +36,37 @@ def standout_mode(option, selected_option)
       selected_option == option ? STANDOUT : NORMAL
     end
 
-    def render_game_text(text, chars_completed, failed = false)
-      current_color = failed ? RED : YELLOW
-      completed = text[0, chars_completed - 1]
-      tail = text[chars_completed + 1, text.length]
-
+    def render_start_game
       self.render do
-        self.setpos(2,4)
-        self.render_multiple_text(GREEN, NORMAL) { completed.each { |c| self.addstr(c) } }
-        self.render_text(c, color, NORMAL)
-        self.render_multiple_text(WHITE, NORMAL) { tail.each { |c| self.addstr(c) } }
+        self.set_render_pos(2, 4)
+        self.render_text("To start playing click ENTER", GREEN, NORMAL)
       end
     end
 
     def render_game_text(text, chars_completed, failed = false)
+      color = failed ? RED : YELLOW
+      head = text[0, chars_completed] || ''
```

Comment:
I used tail in the previous one but we can use completed/remaining to be more clear